scheme/tpm-enacttrust/test/cmd/gen-token/README.md

```diff
@@ -0,0 +1,43 @@
+## Token generation
+
+`gen-token` is used to generate test attestation tokens from JSON descriptions
+of the payload. This is build and used when running the tests.
+
+## Token Format
+
+An attestation token has the following format:
+
+    TPMS_ATTEST_LEN||TPMS_ATTEST||TPMS_ATTEST_SIGNATURE
+
+where
+
+`TPMS_ATTEST_LEN` is a uint16 values containing the length (in bytes) of the
+following `TPMS_ATTEST` structure. `TPMS_ATTEST` is structured according to
+[TPM 2.0
+Specification](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part2_Structures_pub.pdf).
+`TPMS_ATTEST_SIGNATURE` is the ECDSA signature of the `TPMS_ATTEST` structure.
+
+## Token Payload Description Format
+
+`gen-token` takes a path to a file containing the description of the payload
+of the token. This description is a JSON object in the following form:
+
+```json
+{
+  "node-id": "7df7714e-aa04-4638-bcbf-434b1dd720f1",
+  "firmware": 7,
+  "pcrs": [1, 2, 3, 4],
+  "algorithm": 4,
+  "digest": "h0KPxSKAPTEGXnvOPPA/5HUJZjHl4Hu9eg/eYMTPJcc="
+}
+```
+
+note: "signer" and "digest" are decoded as `[]byte`, and so, as per
```

Comment:
Just a clarification question: Where is the "signer" part?

Reply:
erm good point. No idea. I'm guessing it was there at some point, but then was unecessary. Will remove